crates/re_ui/src/command.rs

```diff
@@ -181,6 +183,7 @@ impl Command {
             Command::ToggleCommandPalette => Some(cmd(Key::P)),
 
             Command::PlaybackTogglePlayPause => Some(key(Key::Space)),
+            Command::PlaybackRestart => Some(ctrl_shift(Key::Space)),
```

Comment:
Is there some prior art for what is a good key combination?

On my Mac I find:
* Spotify: `Cmd Left`
* Logic Pro: `Return`
* VLC: `Cmd .`